packages/sources/intrinio/src/util.ts

```diff
@@ -0,0 +1,296 @@
+import * as https from 'https'
+import * as events from 'events'
+import { Logger } from '@chainlink/ea-bootstrap'
+
+const EventEmitter = events.EventEmitter
+
+const SELF_HEAL_BACKOFFS = [0, 100, 500, 1000, 2000]
+
+export class IntrinioRealtime extends EventEmitter {
```

Comment:
We should definitely add integration tests which use something like:
- https://www.npmjs.com/package/@relaycorp/ws-mock
- https://www.npmjs.com/package/mock-socket